css/obBase.css

```diff
@@ -2458,7 +2478,10 @@ div.chatSearchOut .chatSearchIcon {
 .btn-togWrapper {
   border: 1px solid #086A9E;
   border-radius: 5px;
-  float: left;
+  float: right;
+  position: absolute;
```

Comment:
Position absolute overrides float right, if you put position absolute here remove float right.

The other positioning is very specific to this design, it would be better to make a separate class so the original toggle button class is still usable, and the tab-style button has it's own class.